src/roadmap.md

```diff
@@ -103,6 +103,12 @@ one Exonum instance. The necessary set of services should be activated together
 with initialization of the blockchain. This feature serves as a prerequisite for
 dynamic Java services that will be implemented further.
 
+### Java Benchmarking
```

Comment:
That's an internal tool currently, with no plans to open-source anything, AFAIK.

Reply:
What we did, however, is improved considerably the throughput of Java transaction processing (to be released in 0.7), using the developed benchmarking tools to validate the results.

Reply:
We decided to leave this point in the roadmap. Though it is not open-source, it was scheduled and it has been done)